clang/test/SemaSYCL/sycl-esimd-ast.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -sycl-std=2017 -Wno-sycl-2017-compat -ast-dump %s | FileCheck %s
+
+// Tests for AST of sycl_explicit_simd function attribute in SYCL 1.2.1.
```

Comment:
There is no AST test for sycl_explicit_simd . i added this test to show the changes in SYCL 1.2.1 vs SYCL 2020 mode.